### DISCUSSION TITLE
I Have Grown To Resent Callipeg

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#279] [General]
When I started using ToonSquid, I found it a whole lot more enjoyable to use than Callipeg. 

Here's a list of things I don't like about Callipeg compared to ToonSquid: 
- It's overpriced when compared to how feature packed ToonSquid is.
- It's brush selection is very limited compared to ToonSquid.
- ToonSquid has a better brush engine. 
- Feels clunky compared to ToonSquid

Honestly, if anyone were to ask me whether to buy ToonSquid or Callipeg, I'd pick ToonSquid, because I only bought Callipeg when it was the only "affordable" animation app on iPad that had brush smoothing, but now that ToonSquid is here, I'd only suggest buying it if you have money to burn.

**Comment by kavlor** [comment]
I think it's good to have more than one product of similar content as they keep each other in check and create healthy competition on features and pricing.
